readme.md

```diff
@@ -128,9 +128,11 @@ function App() {
 
 #### Holding objects in state without tracking them
 
-This may be useful if you have large, nested objects with accessors that you don't want to proxy. `ref` allows you to keep these objects inside the state model.
+Plain objects and arrays are fully supported with proxy and snapshot. Objects with prototypes may not work for some cases with some unexpected errors. In such cases, you want to bypass tracking with `ref`.
```

Comment:
For me, it still sounds scary that correct behavior can't be ensured. I'd add a more precise reason.

Reply:
Yeah, on second thought, this doesn't add any new insight. I will revert it.